modules/ROOT/pages/manage-versions.adoc

```diff
@@ -64,11 +64,33 @@ If an asset in the development state depends on one or more other assets in the
 From the asset details page in Exchange, manage the implementation versions of a policy asset from the *Manage implementations* table, which displays the latest version. You can view, add, and delete versions: 
 
 . Click *Implementations* from the left navigation bar.
-. To manage the versions of an implementation, click the three vertical dots and select an option:
+. To manage the versions of an implementation, click the more options menu and select an option:
 .. To view the versions, select *Implementation versions*.
 .. To add a new version, select *Add new version* and enter the version.
 .. To delete a version, select *Delete version*. 
 . Click *Close*.
 
+== Update Policy Implementations
+
+To update policy implementations to be compatible with your Flex Gateway version:
+
+. Go to the *Policies* page.
+. Under *Applied policy status*, you'll see the policies that need updating. Click the more options menu icon and click *Check for implementation updates*.
+. A list of all available implementations for your current version shows. +
```

Comment:
Actually, there is no list, users do not choose implementations so what we do is offer the newest compatible version so they can apply it.